src/libs/codepush/options.tsx

```diff
@@ -8,10 +8,12 @@ export const AutoImmediate: CodePushOptions = {
 }
 
 // for production environment, we want a more discreet update
-// On each app start, we check for update, that we install on
-// next app start. This means the app doesn't flicker and the update
+// On each app start/resume, we check for update, that we install on
+// next app start/resume. This means the app doesn't flicker and the update
 // is not visible to the user.
```

Comment:
question(non-blocking):

1. une nouvelle version est disponible
1. on lance l'app
1. on va sur une page spécifique (imaginons, une offre, après avoir fait une longue recherche)
1. on met l'app en background
1. on remet l'app en foreground
1. codepush applique le nouveau bundle


est-ce que dans ce cas, on reste sur la page spécifique ?

sinon :

```suggestion
// next app start/resume.
```

Reply:
1. une nouvelle version est disponible
2. on lance l'app
3. on va sur une page spécifique (imaginons, une offre, après avoir fait une longue recherche)
4. on met l'app en background
5. on remet l'app en foreground
6. codepush applique le nouveau bundle → on voit le splashscreen, puis on arrive sur la home

D'ailleurs, on s'était dit qu'on voulait rajouter le paramètre : `minimumBackgroundDuration` à 5 minutes, pour éviter qu'une fausse manip / le check d'un message, fasse disparaître la page qu'on consultait